community/shadow/APKBUILD

```diff
@@ -3,18 +3,20 @@
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=shadow
 pkgver=4.2.1
-pkgrel=11
+pkgrel=12
 pkgdesc="PAM-using login and passwd utilities (usermod, useradd, ...)"
 url="http://pkg-shadow.alioth.debian.org/"
 arch="all"
 license="GPL"
 depends=""
-makedepends="linux-pam-dev"
-subpackages="$pkgname-doc $pkgname-dbg $pkgname-uidmap"
+makedepends_host="linux-pam-dev"
+subpackages="$pkgname-doc $pkgname-dbg"
+[ ${CBUILD} == ${CHOST} ] && subpackages="$subpackages $pkgname-uidmap"
```

Comment:
`==` is a (unnecessary) bashism, POSIX Command Language specifies only `=` for comparison.

Also it’s better to use `||` in such construct. Some shells don’t recognize such construct as a special case when resulting non-zero status does not mean a failure (when using `set -e`).

And also it’s better to always quote (outside defined) variables, to be defensive against errors.

```sh
[ "$CBUILD" != "$CHOST" ] || subpackages="$subpackages $pkgname-uidmap"
```